brainiak/hyperparamopt/mcmc.py

```diff
@@ -0,0 +1,133 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Metropolis-Hasting Random number generator
+
+This implementation provides random samples from a user-given
```

Comment:
I looked for other python packages that do what I want i.e generate samples from a user-specified pdf. I couldn't find any, so had to write my own. If you know of any other implementation, please let me know as I am not aware of any at this point.